octodns/provider/constellix.py

```diff
@@ -99,14 +102,26 @@ def domain(self, name):
         zone_id = self.domains.get(name, False)
         if not zone_id:
             raise ConstellixClientNotFound()
-        path = '/{}'.format(zone_id)
+        path = '/domains/{}'.format(zone_id)
```

Comment:
Heads up that we're moving towards f-strings now that we're >3.6 so feel free to make that change here if you wish.

/cc https://github.com/octodns/octodns/pull/768